examples/python/easy_start_demo/README.md

```diff
@@ -0,0 +1,28 @@
+## Demo for using gRPC in Python
+
+在Python中使用gRPC时, 进行数据传输的四种方式。(Four ways of data transmission when gRPC is used in Python.)
+
+- #### 简单模式 (unary-unary)
+```text
```

Comment:
Any reason to put the body in a monospace font? This is generally used for code snippets.